### PR TITLE
Added TopQuadrant Validator

### PR DESCRIPTION
**Pull request by umbreak**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -123,7 +123,7 @@ lazy val kg = project
       serviceKafka,
       serviceKamon,
       serviceHttp,
-      wesoSchema,
+      topQuadrantShacl,
```
Comment (bogdanromanx) [inline build.sbt]: should also remove `resolvers  += Resolver.bintrayRepo("bogdanromanx", "maven")` as that's no longer required.

File: src/main/resources/logback.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
```
Comment (bogdanromanx) [inline src/main/resources/logback.xml]: what does this do?

File: src/main/scala/ch/epfl/bluebrain/nexus/kg/resources/Rejection.scala [hunk 3]
```diff
@@ -137,7 +137,7 @@ object Rejection {
     * @param schema a reference to the schema
     * @param report the validation report
     */
-  final case class InvalidResource(schema: Ref, report: ValidationReport)
+  final case class InvalidResource(schema: Ref, report: Json)
```
Comment (bogdanromanx) [inline src/main/scala/ch/epfl/bluebrain/nexus/kg/resources/Rejection.scala]: could we preserve the validation report and handle the json conversion at the routes level?

File: src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/ResourceRoutes.scala [hunk 4]
```diff
@@ -208,7 +203,7 @@ class ResourceRoutes(resources: Resources[Task])(implicit cache: DistributedCach
             }
 
           case `viewSchemaUri` =>
-            trace("listResolvers") {
+            trace("listViews") {
```
Comment (bogdanromanx) [inline src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/ResourceRoutes.scala]: :metal:

File: src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/ResourceRoutes.scala [hunk 5]
```diff
@@ -266,7 +257,7 @@ class ResourceRoutes(resources: Resources[Task])(implicit cache: DistributedCach
       }
     } ~
       // add tag to resource
-      (evalBool(withTag) & pathPrefix("tags") & projectNotDeprecated) {
```
Comment (bogdanromanx) [inline src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/ResourceRoutes.scala]: :metal:

File: src/main/resources/logback.xml (hunk 2 above)
Comment (umbreak) [reply src/main/resources/logback.xml]: https://stackoverflow.com/questions/3257154/how-to-prevent-logback-from-outputting-its-own-status-at-the-start-of-every-log

**Pull request merged**